api/controllers/FeedbackController.ts

```diff
@@ -14,9 +13,12 @@ import {
 @UseBefore(UserAuthentication)
 @JsonController('/feedback')
 export class FeedbackController {
-  @Inject()
   private feedbackService: FeedbackService;
 
+  constructor(feedbackService: FeedbackService) {
```

Comment:
We don't need to inject the service in the constructor? I see in the typedi and routing-controllers docs that we don't need the `@Inject` if the class has the `@Service` annotation, but I don't see anything like that for controller annotations.

Reply:
Looks like we don't need them - in fact, including them throws an error when sending the controller a request:
`ServiceNotFoundError: Cannot determine a class of the requesting service \"undefined\"`